### PR TITLE
[RFR] Yehoyachin!

### PR DESCRIPTION
**Pull request by pavelzag**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!flask/bin/python
+from flask import Flask
+
+
+app = Flask(__name__)
+
+
+@app.route('/yehoyachin')
+def yehoyachin():
```
Comment (gshefer) [inline app.py]: Please add description to the function

File: app.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    return 'yehoyachin!'
+
+if __name__ == '__main__':
+    app.run(debug=True, use_reloader=True)
```
Comment (gshefer) [inline app.py]: Koko moko

**Comment by gshefer** [comment]
Please add some description to the pull request.

**Comment by gshefer** [comment]
Please add a state to the PR title - setting state as [WIP].

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
Pull request is inactive for 22 days and 6 hours- please do some action - update it or close it

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
@pavelzag @gshefer your email addresses are missing, please set it up (https://help.github.com/articles/verifying-your-email-address/)

**Comment by RHnudgeBot0** [comment]
@pavelzag @gshefer your email addresses are missing, please set it up (https://help.github.com/articles/verifying-your-email-address/)

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
@pavelzag @gshefer your email addresses are missing, please set it up (https://help.github.com/articles/verifying-your-email-address/)

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
@pavelzag @gshefer your email addresses are missing, please set it up (https://help.github.com/articles/verifying-your-email-address/)

**Comment by RHnudgeBot0** [comment]
Please add some description to the pull request.

**Comment by RHnudgeBot0** [comment]
@pavelzag @gshefer your email addresses are missing, please set it up (https://help.github.com/articles/verifying-your-email-address/)

**Pull request closed without merging**